src/test/java/uk/gov/companieshouse/filinghistory/consumer/mapper/pretransform/DeltaNodeMapperTest.java

```diff
@@ -0,0 +1,272 @@
+package uk.gov.companieshouse.filinghistory.consumer.mapper.pretransform;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.delta.DescriptionValues;
+import uk.gov.companieshouse.api.delta.FilingHistory;
+import uk.gov.companieshouse.api.delta.FilingHistoryDelta;
+import uk.gov.companieshouse.filinghistory.consumer.transformrules.TransformerTestingUtils;
+import uk.gov.companieshouse.filinghistory.consumer.transformrules.functions.FormatDate;
+
+@ExtendWith(MockitoExtension.class)
+class DeltaNodeMapperTest {
+
+    private static final ObjectMapper objectMapper = TransformerTestingUtils.getMapper();
+
+    private DeltaNodeMapper preTransformMapper;
+    @Mock
+    private FormatDate formatDate;
+
+    @BeforeEach
+    void setUp() {
+        preTransformMapper = new DeltaNodeMapper(objectMapper, formatDate);
```

Comment:
Do we need this? The value can be assigned once in the declaration, assuming the mapper is stateless

Reply:
@cgregoryCH explained that this fixes a mock ordering issue